### PR TITLE
Translations: Bugfix, drop columns, new defaults

### PR DESCRIPTION
**Pull request by matentzn**
This PR:

- Changes a few default values for the new international edition
- Adds a new feature to drop unsupported columns during the preparation process
- Fixes a wrong references in the owl conversion of the translations

### CODE REVIEW

**Review by gouttegd — APPROVED** [review]
I approve, but please see my remarks about the use of a string-typed parameter in the config file (both for the newly added `drop_unknown_columns` and the other parameters added in #1006). They should really be boolean parameters.

(I am not _requesting_ any change since it’s my fault for not having caught this mistyping before. Your call whether you want to change the type or not.)

File: odk/odk.py [hunk 1]
```diff
@@ -213,9 +213,12 @@ class BabelonTranslationProduct(Product):
     include_not_translated: str = "false"
     """if include_not_translated is 'false' NOT_TRANSLATED values are removed during preprocessing."""
     
-    update_translation_status: str = "false"
+    update_translation_status: str = "true"
```
[the remark this answers is not in the document]
Comment (matentzn) [reply odk/odk.py]: Did it, I forgot about `lower`!

File: template/src/ontology/Makefile.jinja2 [hunk 2]
```diff
@@ -962,6 +962,7 @@ $(TRANSLATIONSDIR)/{{ translation.id }}-preprocessed.babelon.tsv: $(TRANSLATIONS
 		--output-not-translated $(TRANSLATIONSDIR)/{{ translation.id }}-not-translated.babelon.tsv \
 		--include-not-translated {{ translation.include_not_translated|default('false') }} \
 		--update-translation-status {{ translation.update_translation_status|default('true') }} \
+		--drop-unknown-columns {{ translation.drop_unknown_columns|default('true') }} \
```
[the remark this answers is not in the document]
Comment (matentzn) [reply template/src/ontology/Makefile.jinja2]: Ahhhh, forgot about this, thanks. Done!

**Comment by matentzn** [comment]
Thank you, I also tested your suggestion and it works perfect.

File: odk/odk.py [hunk 3]
```diff
@@ -210,13 +210,13 @@ class BabelonTranslationProduct(Product):
     language: str = "en"
     """Language tag (IANA/ISO), e.g 'en', 'fr'."""
     
-    include_not_translated: str = "false"
+    include_not_translated: str = False
```
Comment (gouttegd) [inline odk/odk.py:213]: These should be declared as `bool`, not `str`:

```python
include_not_translated: bool = False
```

**Review by gouttegd — CHANGES_REQUESTED** [review]
Glad that you agreed with the proposed change, but then it should be done correctly. It’s not enough to assign a default value of `True` or `False` – the option itself needs to be declared as `bool`. Otherwise you’re merely declaring a string-typed option that happens to have a default value of `True` or `False`.

File: odk/odk.py (hunk 3 above)
Comment (matentzn) [reply odk/odk.py]: oops. weird it worked without complaints

**Review by gouttegd — APPROVED** [review]

**Pull request merged**